### DISCUSSION TITLE
Closing issues without explanation

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#35072] [General] [answered]
PLEASE, PLEASE dont just close pull requests without any explanation. It's not only rude and disrespectful to the people who contributed their time to submit a pull request but without explanation its unknown why.

Joomla 3.10 will be supported for 24 months and several of the closed pull requests tonight are joomla 3 bug fixes

Other pull requests are perfectly valid "new features" so they should be flagged as "needs updating/replacing for j4" and kept open until someone has done that.

**Comment by HLeithner** [comment, marked as answer]
It's not intended, seems something went wrong on github branch rename feature. We are looking into it.

**Reply by brianteeman** [reply]
ok - thanks

**Reply by HLeithner** [reply]
The reason is that the repositories owned by the PR authors were deleted already, so github can't rebase the pr...

**Reply by richard67** [reply]
11 PRs have been closed for that reason by GitHub. We have a list and will check them one by one,
